proto/v1/review_service.proto

```diff
@@ -130,6 +137,12 @@ message ApproveReviewRequest {
   string name = 1 [(google.api.field_behavior) = REQUIRED];
 }
 
+message SendBackReviewRequest {
+  // The name of the review to add an approver.
+  // Format: projects/{project}/reviews/{review}
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
shall we add a field sth like `comment` or `reason` and attach the value to the auto-generated activity?
Same things may happen to `ApproveReview(ApproveReviewRequest)`

Reply:
SGTM. Will do in a following PR.